### PR TITLE
Demo_collecting posts

### PR DESCRIPTION
**Pull request by yoouung**
- `1_online_mall_review`, `2_free_board` : demo version shell code
- `2_free_board/collecting_posts.py` : draft code of collecting posts

### CODE REVIEW

File: demo/1_online_mall_review/1_duplication_output.csv
Comment (doorooful) [inline demo/1_online_mall_review/1_duplication_output.csv:1]: Could you declare output file on `.gitignore` file?

File: demo/1_online_mall_review/2_typo_output.csv
Comment (doorooful) [inline demo/1_online_mall_review/2_typo_output.csv:1]: Same here!

File: demo/1_online_mall_review/4_pdd.py
Comment (doorooful) [inline demo/1_online_mall_review/4_pdd.py:1]: I think pdd is needed 👍

File: typo/typo.py [hunk 1]
```diff
@@ -1,22 +1,25 @@
 import requests
 import json
-import os # TODO: remove
+import os  # TODO: remove
 import argparse
 
+
```
Comment (doorooful) [inline typo/typo.py:6]: Could you remove this line?

**Comment by doorooful** [comment]
Please review the comments for the reference ◡̈

File: typo/typo.py (hunk 1 above)
Comment (yoouung) [reply typo/typo.py:6]: As I delete the lines and save the code files, the autopop8 extension makes again these empty lines, maybe for python formatting.

**Pull request closed without merging**